Neos.Fusion/Classes/Core/Parser.php

```diff
@@ -25,7 +25,8 @@
 /**
  * The Fusion Parser
  *
- * @api
+ * @internal The Fusion parsing is considered internal.
+ *           For interacting with Fusion from the outside a FusionView should be used.
```

Comment:
the parser being api was introduced via this commit: https://github.com/neos/neos-development-collection/commit/fde5445127f83a3ca8a1d2c2cc6ea3fc6fcc07e8

But if the runtime should not be api (which it should not) one wont need the parser to be api :)

TODO: Fix api annotation in `parseFromSource`